pkg/kubevirt/util.go

```diff
@@ -0,0 +1,33 @@
+package kubevirt
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// KubevirtClient creates kubevirt client based on the kubeconfig of the kubevirt cluster which is saved in the secret that
+// is passed to it.
+func KubevirtClient(secret *v1.Secret) (client.Client, error) {
```

Comment:
This function is not used anywhere. Could you please add the code that would use it, or at least clarify your intention for adding it?

Reply:
As discussed earlier this will be used once we decide how we will create those Data Volumes, as this function will be passed to the constructor of the manager.

Reply:
Thanks for clarifying. Also, as discussed today, the namespace to be used should be the one specified in the secret, so please return it as a second result from this function and remove it from the parameters of the `DataVolumeManager` interface.